wordpress-mobile/WordPress-Android
# [Reader Improvements] X-post and P2 card improvements

Part of #19082

This implements the new specs for x-posts (ref: zQnohyMpLzBzQ5jzMMKni3-fi-1348%3A11005) and also updates the avatar section of the default post card for P2 posts, to match the X-Post avatar positions.

To test:
1. Install and log in to Jetpack app
2. Make sure the `ReaderImprovementsFeatureConfig` is disabled in debug settings
3. Go to a P2 site page that has X-Posts in the Reader
4. **Verify** the post card UI for P2 posts and X-Posts did not change compared to previous versions
5. Enable the `ReaderImprovementsFeatureConfig` in debug settings
6. Go to a P2 site page that has X-Posts in the Reader
7. **Verify** the post card UI for P2 posts and X-Posts matches the specs

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A (UI only)

PR submission checklist:


UI Changes testing checklist:
